src/Drupal/Driver/Cores/CoreInterface.php

```diff
@@ -216,4 +216,14 @@ public function configGet($name, $key = '');
    */
   public function configSet($name, $key, $value);
 
+  /**
+   * Create an entity.
+   */
+  public function entityCreate($entity_type, $entity);
+
+  /**
+   * Delete an entity.
+   */
+  public function entityDelete($entity_type, $entity);
```

Comment:
I hadn't realized most other methods in here were missing the `@param` as well, so I think what you have is good to go.